app.py

```diff
@@ -1,4 +1,4 @@
-from app_creator import create_app
+from factory import create_app
```

Comment:
factory seems less informative than app_creator? can you explain this choice? will you stick every creation in this file?

Reply:
As of now, nothing else is created with that file. `app_creator` is better.